sqlite-db/prepare_db.sh

```diff
@@ -15,4 +15,4 @@ trap 'rm -f $TEMPDB' EXIT
 DATABASE_URL=sqlite:$TEMPDB cargo sqlx migrate run
 
 # prepare the sqlx-data.json rust mappings
-DATABASE_URL=sqlite:./$DAEMON_DIR/$TEMPDB SQLX_OFFLINE=true cargo sqlx prepare
+DATABASE_URL=sqlite:./$CRATE_DIR/$TEMPDB SQLX_OFFLINE=true cargo sqlx prepare --merged -- --all-targets
```

Comment:
❓What does this do `--merged -- --all-targets` do?

Reply:
Oh, this was a rebase error. It was originally added by @luckysori  and lateron removed. 
Thanks for the good eyes.